shell/src/chain_manager.rs

```diff
@@ -631,6 +633,38 @@ impl ChainManager {
                             }.into(),
                             topic: ShellChannelTopic::ShellEvents.into(),
                         }, Some(ctx.myself().into()));
+
+                    // handle operations
+                    if inject_data.block_header.level() > 1 {
```

Comment:
@adonagy 
can you pls. add here more comments, about whats going on inside this IF and why it is needed?